src/test/scala/nl.knaw.dans.easy.bagstore/component/BagStoreSpec.scala

```diff
@@ -167,4 +169,18 @@ class BagStoreSpec extends TestSupportFixture
     Files.getPosixFilePermissions(bagInStore.resolve("data")) shouldBe dirPermissions
     Files.getPosixFilePermissions(bagInStore.resolve("data/sub")) shouldBe dirPermissions
   }
+
+  it should "result in a failure when the fetch.txt is invalid" in {
+    val uuid1 = UUID.fromString("00000000-0000-0000-0000-000000000001")
+    val fetchFile = File(testBagPrunedB.resolve("fetch.txt"))
+
+    Files.write(fetchFile.path, fetchFile.contentAsString.replaceAll("01", "10").getBytes())
+
+    bagStore.add(testBagPrunedB, Some(uuid1)) should matchPattern {
```

Comment:
this is a bit of a weird test IMHO. Shouldn't it be something like:

```scala
class BagStoreSpec extends TestSupportFixture
  with OneInstancePerTest
  with Inspectors // new import! necessary for `forEvery` below
  with ...

// lots of tests

inside(bagStore.add(testBagPrunedB, Some(uuid1))) {
  case Failure(CompositeException(errors)) =>
    forEvery(errors)(e => {
      e shouldBe an[IllegalArgumentException]
      e.getMessage should include ("Local-file-uri found in fetch.txt can not be found in the bag-store: ")
    })
}
```

Reply:
Nice, i was not aware of the forEvery function